exporters/aerospike/aerospike.prometheus.json.tmpl

```diff
@@ -24,6 +24,7 @@
                     {{/* Add below any specific rules for this template. These will be hardcoded. */}}
                     {"add_attributes":[{"prefix":"aerospike_","attributes":{"aerospike_cluster_name": "{{.config.aerospike_cluster_name}}" }}]}
                 ],
+                "scrape_timeout": "5s",
```

Comment:
I believe you copied the wrong line from the PR...
it should be configurable, therefore something like:
```
                {{- if .config.scrape_timeout }}
                "scrape_timeout": "{{.config.scrape_timeout}}",
                {{ end }}
```

Reply:
oh.. thank you @paologallinaharbur , I have updated it. 👍